src/dev/code_coverage/shell_scripts/copy_mocha_reports.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+EXTRACT_START_DIR=tmp/extracted_coverage
```

Comment:
(sorry, accidentally deleted my question I meant to link here)

You have `tmp/extracted_coverage` defined here, but `/tmp/extracted_coverage` in `extract_archives.sh`. Should be differ?

Reply:
It's concatenated: https://github.com/wayneseymour/kibana/blob/579dcc1903560dfcecaf875788823d5ece77daf1/src/dev/code_coverage/shell_scripts/copy_mocha_reports.sh#L5